spec/base-spec.js

```diff
@@ -0,0 +1,43 @@
+/** @babel */
+
+/* This file contains all specs to ensure the base-functionality of
+this plugin. */
+
+import _path from 'path';
+// _helpers = require(_path.join(__dirname, '_spec-helpers.coffee'));
```

Comment:
It's still showing as here for me, unless you mean you removed it and haven't pushed the commit? :stuck_out_tongue: